project/plugins.sbt

```diff
@@ -11,7 +11,9 @@ addSbtPlugin("com.typesafe.sbt" % "sbt-site" % "0.7.1")
 
 addSbtPlugin("com.typesafe.sbt" % "sbt-osgi" % "0.6.0")
 
-addSbtPlugin("com.typesafe" % "sbt-mima-plugin" % "0.1.6")
+addSbtPlugin("com.typesafe" % "sbt-mima-plugin" % "0.1-b71a3e88f0a21d6ce5efe7c0580dba9784295358")
+
```

Comment:
good catch, yes it is.
I made a separate bump PR, https://github.com/akka/akka/pull/17605 so if we include the bump here the other one can be closed